modules/modules.nix

```diff
@@ -157,6 +157,7 @@ let
     ./programs/skim.nix
     ./programs/sm64ex.nix
     ./programs/sqls.nix
+    ./programs/swiftbar.nix
```

Comment:
Should this be guarded by an `isDarwin` optional? The module itself asserts the platform is Darwin already, but I'm not sure that would cause the manpage entries to not get generated on Linux.